waku/v2/node/peer_manager/peer_manager.nim

```diff
@@ -45,7 +45,7 @@ const
   MaxParalelDials = 10
 
   # Delay between consecutive relayConnectivityLoop runs
-  ConnectivityLoopInterval = chronos.seconds(30)
+  ConnectivityLoopInterval = chronos.seconds(15)
```

Comment:
Any particular reason for this change?

Reply:
yep, its intended. 30 seconds was to much.
since right now we have the backoff, it doesnt hurt to run it more often, since we no longer spam connections